packages/psss-server/src/connections/ReportConnection.ts

```diff
@@ -9,4 +9,15 @@ const { REPORT_API_URL = 'http://localhost:8030/v2' } = process.env;
 
 export class ReportConnection extends ApiConnection {
   baseUrl = REPORT_API_URL;
+
+  async fetchReport(reportCode: string, orgUnitCodes: string[], periods: string[] = []) {
+    if (!orgUnitCodes || !orgUnitCodes.length) {
+      throw new Error(`No organisationUnitCodes provided`);
+    }
+
+    return this.post(`fetchReport/${reportCode}`, {
+      organisationUnitCodes: orgUnitCodes.join(','),
+      period: periods.join(';')
```

Comment:
So the report server expects period to be specified in the same way as `DHIS2`, rather than as `startDate` and `endDate`?

Reply:
Yep from what I understand

Reply:
Oh right! Cool...I guess that can complicate things, but might be needed to support specific use cases.

Reply:
Related: https://github.com/beyondessential/tupaia/pull/1645#discussion_r527298489